django/urls/resolvers.py

```diff
@@ -331,6 +339,19 @@ def check(self):
             )
         return warnings
 
+    def _check_pattern_unmatched_angle_brackets(self):
+        match = re.match(_PATH_ANGLE_BRACKET_COMPONENT_RE, str(self._route))
+        if not match:
+            warning = Warning(
+                "Your URL pattern {} has a route that contains `<` or '>', "
+                "begins with a `<` or ends with a '>', but is not matched.".format(
+                    self.describe()
```

Comment:
This message is too wordy. I suggest:

> URL pattern {} has an unmatched {} bracket

We can adjust to include *which* kind of bracket is unmatched in the second format parameter there. This can be determined from the `opened` variable that I suggested above. If `opened` is true, `<` is unmatched, otherwise `>`.